sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -1036,3 +1036,18 @@ def test_companion():
         Matrix([[0, -c0], [1, -c1]])
     assert Matrix.companion(Poly([1, c2, c1, c0], x)) == \
         Matrix([[0, 0, -c0], [1, 0, -c1], [0, 1, -c2]])
+
+
+def test_rmul_pr19860():
+    class Foo(ImmutableDenseMatrix):
+        _op_priority = 11.01
```

Comment:
It would be better to use something like `ImmutableDenseMatrix._op_priority + .01`

Reply:
Fixed